redis/src/main/scala/zio/redis/options/Streams.scala

```diff
@@ -0,0 +1,65 @@
+package zio.redis.options
+
+import zio.duration._
+
+trait Streams {
+  sealed case class Idle(millis: Long)
+
+  sealed case class Time(millis: Long)
+
+  sealed case class RetryCount(count: Long)
+
+  case object WithForce {
+    private[redis] def stringify = "FORCE"
+  }
+
+  type WithForce = WithForce.type
+
+  case object WithJustId {
+    private[redis] def stringify = "JUSTID"
+  }
+
+  type WithJustId = WithJustId.type
+
+  sealed trait XGroupCommand
+
+  object XGroupCommand {
+    case class Create(key: String, group: String, id: String, mkStream: Boolean) extends XGroupCommand
```

Comment:
Make case classes final.

Reply:
This won't work for older Scala versions.